lib/Swagger/src/ContentEntity/Api/UserToEntityRelationshipsApi.php

```diff
@@ -172,6 +174,11 @@ public function createWithHttpInfo($entity_id, $user_id)
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)